plugins/shortcuts/src/ShortcutItem.tsx

```diff
@@ -94,6 +117,14 @@ export const ShortcutItem = ({ shortcut, api, allowExternalLinks }: Props) => {
           >
             <EditIcon className={classes.icon} />
           </IconButton>
+          <IconButton
```

Comment:
I think that like @ahhhndre said, we should make this opt in. Theres not a lot of room there in the first place.

Reply:
yes !! made it opt in working fine please review